Lib/copyreg.py

```diff
@@ -36,6 +36,11 @@ def pickle_union(obj):
 
 pickle(type(int | str), pickle_union)
 
+def pickle_super(obj):
```

Comment:
Should we give this a private name? All the existing ones also use public names, so I don't mind keeping this one public too.